src/app/component/CmsBlock/CmsBlocks.style.scss

```diff
@@ -0,0 +1,23 @@
+/**
+ * ScandiPWA - Progressive Web App for Magento
+ *
+ * Copyright © Scandiweb, Inc. All rights reserved.
+ * See LICENSE for license details.
+ *
+ * @license OSL-3.0 (Open Software License ("OSL") v. 3.0)
+ * @package scandipwa/base-theme
+ * @link https://github.com/scandipwa/base-theme
+ */
+
+.CmsBlock {
+    &-Wrapper{
+        .Image {
```

Comment:
Please try implementing this via the ratio of the image, calculate a custom padding-bottom. If image will be missing, the block will still take desired place.